packages/config/src/discovery/values/projects/zksync2/ZKSYNC_2.ts

```diff
@@ -0,0 +1,12 @@
+import { ProjectRiskViewEntry } from '../../../../common'
+
+const UPGRADEABILITY_STRING: ProjectRiskViewEntry = {
+  value: 'Yes',
+  description:
+    'The code that secures the system can be instantly upgraded without any notice.',
+  sentiment: 'bad',
+}
```

Comment:
How does this differ from RISK_VIEW.UPGRADABLE_YES? We should be really careful while creating custom risk per project. This can cause some differences in how we categorise the same risk.